src/test/java/com/google/sps/ProjectServletTestUtils.java

```diff
@@ -0,0 +1,77 @@
+  public static final String expectedGlobalOnly = "[\n  {\n    \"name\": \"MyProject3\",\n    \"timestamp\": \"2020-08-12T05:39:02.385Z\",\n    \"visibility\": \"public\",\n    \"owners\": [\n      \"xyz@xyz.com\"\n    ],\n    \"editors\": [\n      \"xyz@abc.com\",\n      \"uvw@def.com\"\n    ]\n  },\n  {\n    \"name\": \"MyProject2\",\n    \"timestamp\": \"2020-08-12T05:39:02.384Z\",\n    \"visibility\": \"public\",\n    \"owners\": [\n      \"abc@xyz.com\"\n    ],\n    \"editors\": [\n      \"uvw@def.com\"\n    ]\n  }\n]\n";
+  public static final String expectedSortAsc = "[\n  {\n    \"name\": \"MyProject\",\n    \"timestamp\": \"2020-08-12T05:39:02.383Z\",\n    \"visibility\": \"private\",\n    \"owners\": [\n      \"abc@xyz.com\",\n      \"def@uvw.com\"\n    ],\n    \"editors\": [\n      \"xyz@abc.com\",\n      \"uvw@def.com\"\n    ]\n  },\n  {\n    \"name\": \"MyProject2\",\n    \"timestamp\": \"2020-08-12T05:39:02.384Z\",\n    \"visibility\": \"public\",\n    \"owners\": [\n      \"abc@xyz.com\"\n    ],\n    \"editors\": [\n      \"uvw@def.com\"\n    ]\n  },\n  {\n    \"name\": \"MyProject3\",\n    \"timestamp\": \"2020-08-12T05:39:02.387Z\",\n    \"visibility\": \"private\",\n    \"owners\": [\n      \"xyz@xyz.com\"\n    ],\n    \"editors\": [\n      \"abc@xyz.com\"\n    ]\n  }\n]\n";
+  public static final String expectedSpecificProject = "[\n  {\n    \"name\": \"MyProject\",\n    \"timestamp\": \"2020-08-12T05:39:02.383Z\",\n    \"visibility\": \"private\",\n    \"owners\": [\n      \"abc@xyz.com\",\n      \"def@uvw.com\"\n    ],\n    \"editors\": [\n      \"xyz@abc.com\",\n      \"uvw@def.com\"\n    ]\n  }\n]\n";
+  public static final String expectedRoleEditor = "[\n  {\n    \"name\": \"MyProject3\",\n    \"timestamp\": \"2020-08-12T05:39:02.387Z\",\n    \"visibility\": \"private\",\n    \"owners\": [\n      \"xyz@xyz.com\"\n    ],\n    \"editors\": [\n      \"abc@xyz.com\"\n    ]\n  }\n]\n";
+  public static final String expectedRoleOwner = "[\n  {\n    \"name\": \"MyProject2\",\n    \"timestamp\": \"2020-08-12T05:39:02.384Z\",\n    \"visibility\": \"public\",\n    \"owners\": [\n      \"abc@xyz.com\"\n    ],\n    \"editors\": [\n      \"uvw@def.com\"\n    ]\n  },\n  {\n    \"name\": \"MyProject\",\n    \"timestamp\": \"2020-08-12T05:39:02.383Z\",\n    \"visibility\": \"private\",\n    \"owners\": [\n      \"abc@xyz.com\",\n      \"def@uvw.com\"\n    ],\n    \"editors\": [\n      \"xyz@abc.com\",\n      \"uvw@def.com\"\n    ]\n  }\n]\n";
+  public static final String expectedSearchTerm = "[\n  {\n    \"name\": \"MyProject\",\n    \"timestamp\": \"2020-08-12T05:39:02.383Z\",\n    \"visibility\": \"private\",\n    \"owners\": [\n      \"abc@xyz.com\",\n      \"def@uvw.com\"\n    ],\n    \"editors\": [\n      \"xyz@abc.com\",\n      \"uvw@def.com\"\n    ]\n  }\n]\n";
```

Comment:
Is there a more maintainable way to have these expected Json strings? 

For example, can we have a java class that we then convert to json using gson?